x/ccv/consumer/keeper/distribution.go

```diff
@@ -103,62 +103,74 @@ func (k Keeper) shouldSendRewardsToProvider(ctx sdk.Context) bool {
 // all the block rewards allocated for the provider
 func (k Keeper) SendRewardsToProvider(ctx sdk.Context) error {
 	// empty out the toSendToProviderTokens address
-	ch := k.GetDistributionTransmissionChannel(ctx)
-	transferChannel, found := k.channelKeeper.GetChannel(ctx, transfertypes.PortID, ch)
-	if found && transferChannel.State == channeltypes.OPEN {
-		tstProviderAddr := k.authKeeper.GetModuleAccount(ctx,
-			types.ConsumerToSendToProviderName).GetAddress()
-		providerAddr := k.GetProviderFeePoolAddrStr(ctx)
-		timeoutHeight := clienttypes.ZeroHeight()
-		transferTimeoutPeriod := k.GetTransferTimeoutPeriod(ctx)
-		timeoutTimestamp := uint64(ctx.BlockTime().Add(transferTimeoutPeriod).UnixNano())
-
-		sentCoins := sdk.NewCoins()
-		var allBalances sdk.Coins
-		// iterate over all whitelisted reward denoms
-		for _, denom := range k.AllowedRewardDenoms(ctx) {
-			// get the balance of the denom in the toSendToProviderTokens address
-			balance := k.bankKeeper.GetBalance(ctx, tstProviderAddr, denom)
-			allBalances = allBalances.Add(balance)
-
-			// if the balance is not zero,
-			if !balance.IsZero() {
-				packetTransfer := &transfertypes.MsgTransfer{
-					SourcePort:       transfertypes.PortID,
-					SourceChannel:    ch,
-					Token:            balance,
-					Sender:           tstProviderAddr.String(), // consumer address to send from
-					Receiver:         providerAddr,             // provider fee pool address to send to
-					TimeoutHeight:    timeoutHeight,            // timeout height disabled
-					TimeoutTimestamp: timeoutTimestamp,
-					Memo:             "consumer chain rewards distribution",
-				}
-				_, err := k.ibcTransferKeeper.Transfer(ctx, packetTransfer)
-				if err != nil {
-					return err
-				}
-				sentCoins = sentCoins.Add(balance)
+	sourceChannelID := k.GetDistributionTransmissionChannel(ctx)
+	transferChannel, found := k.channelKeeper.GetChannel(ctx, transfertypes.PortID, sourceChannelID)
+	if !found || transferChannel.State != channeltypes.OPEN {
+		k.Logger(ctx).Info("WARNING: cannot send rewards to provider;",
+			"transmission channel not in OPEN state", "channelID", sourceChannelID)
+		return nil
```

Comment:
Do we want to return an `err != nil` or maybe log an `Error` instead of `Info`. It seems that when we cannot [send the rewards](https://github.com/cosmos/interchain-security/blob/45c7e360fd8ffc3d312230822f514077705668f6/x/ccv/consumer/keeper/distribution.go#L34) we log a message with a level `Error`.

Reply:
This was the current behavior and I wouldn't want to change it as it works. I think it's fine to log a warning in case this happens. Since we don't return an error, it's weird to log an error.